srcjs/bindings/panelInputBinding.js

```diff
@@ -20,6 +20,13 @@ $.extend(f7PanelBinding, {
         $(el).trigger('shown');
       }
     };
+
+    // if parent class is split-layout, add visibleBreakpoint
+    // makes the left panel always visible on tablets and desktop
+    if ($(el).parent().hasClass('split-layout')) {
+      config.visibleBreakpoint = 1024;
+    }
```

Comment:
@DivadNojnarg I think this is an elegant solution to get a split layout on wide enough screens. Prevents a lot of custom styles. One thing I do find annoying is that when the left panel cannot be collapsed, the burger icon still is there, but you can't click on it (that's a Framework7 thing).

We could also give the user control over `1024` by only setting the visibleBreakpoint if it has not been given when using split layout:

```
    if ($(el).parent().hasClass('split-layout')) {
      if (!config.visibleBreakpoint) {
        config.visibleBreakpoint = 1024;
      }
    }
```